cmd/empire/main.go

```diff
@@ -82,6 +88,12 @@ var Commands = []cli.Command{
 		Usage:     "Run the empire HTTP api",
 		Flags: append([]cli.Flag{
 			cli.StringFlag{
+				Name:   FlagURL,
+				Value:  "",
+				Usage:  "That base URL where this Empire instance runs",
```

Comment:
It's only required if the `saml` authentication backend is used. I'll update this to error out if the `saml` backend is used, but this isn't provided.